test/dlc_tests/sanity/test_pt_s3.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from invoke.context import Context
+from packaging.version import Version
+
+from test.test_utils import (
+    get_container_name,
+    get_framework_and_version_from_tag,
+    get_processor_from_image_uri,
+    run_cmd_on_container,
+    start_container,
+)
+
+
+@pytest.mark.model("N/A")
+def test_pt_s3_sanity(pytorch_training):
```

Comment:
Please add the `integration` marker as well.

Reply:
done